returnn/tf/util/basic.py

```diff
@@ -1339,6 +1340,142 @@ def __call__(self, shape, dtype=None, partition_info=None):
   return LoadTxtFileInitializer()
 
 
+def greenwood_function(x, scaling_constant=165.4, constant_of_integration=0.88, slope=2.1):
+  """
+  Greenwood function, convert fractional length to frequency, see
+  Schlueter, Ralf, et al. "Gammatone features and feature combination for large vocabulary speech recognition."
+  ICASSP 2007 and also
+  https://en.wikipedia.org/wiki/Greenwood_function
+  The default values are taken from RASR, see https://github.com/rwth-i6/rasr/blob/master/src/Signal/GammaTone.cc
```

Comment:
Add a new line after your description, before the first param.

About the default values, is there any reason they are like this? (Also in RASR?)

Reply:
They are apparently recommended for this model when applied to human data. Both Ralf's paper and Wikipedia cite the paper by Greenwood which I added in the comment now, but which is not publicly accessible, unfortunately.